dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -132,6 +131,19 @@ RUN apt-get update                  \
     && apt-get update                                                                                              \
     && apt-get install -y docker-ce-cli
 
+# Install Azure CLI, following https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt#option-2-step-by-step-installation-instructions
+RUN apt-get update                           \
+    &&apt-get install -y ca-certificates     \
+                       curl                  \
+                       apt-transport-https   \
+                       lsb-release gnupg
```

Comment:
Can you take this chance to move installation of the dependencies to the top `apt-get install`?

Reply:
refined